cm.py

```diff
@@ -180,8 +180,13 @@ def failure(timeout):
                            time_between_checks=3, timeout=600, success=success, failure=failure)
 
     def stop(self):
-        command_id = self.api_client.stop_all_cluster_services(cluster_name=self.name)['id']
-        _wait_for_command(self, command_id)
+        services = self.api_client.get_cluster_services(cluster_name=self.name,
+                                                        view='summary')['items']
+        for service in services:
```

Comment:
Can you add a comment explaining why you're doing this? Also, you may want to use a `break` in the loop instead of a `return` just to make it clear that you're just trying to stop the loop (and not return anything).